lib/daru.rb

```diff
@@ -26,19 +26,20 @@ module Daru
     10 => 31,
     11 => 30,
     12 => 31
-  }
+  }.freeze
 
-  SPLIT_TOKEN = ','
-  class << self
-    @@lazy_update = false
+  @lazy_update = false
```

Comment:
With my currenty `@lazy_update` class instance variable branch:
```
2.2.4 :001 > Daru.lazy_update
 => false 
2.2.4 :002 > Daru.lazy_update = true
 => true 
2.2.4 :003 > Daru.lazy_update
 => true 
```

I assume, that what I changed, should still work.